spec/text-editor-spec.coffee

```diff
@@ -916,6 +916,12 @@ describe "TextEditor", ->
         editor.setCursorBufferPosition([3, 1])
         expect(editor.getCurrentParagraphBufferRange()).toBeUndefined()
 
+    describe "getCursorAtScreenPosition(screenPosition)", ->
+      it "returns the cursor at the given screenPosition", ->
+        cursor1 = editor.addCursorAtScreenPosition([0, 2])
+        cursor2 = editor.getCursorAtScreenPosition(cursor1.getScreenPosition())
+        expect(cursor2.marker).toBe cursor1.marker
```

Comment:
I would think that `expect(cursor2).toBe(cursor1)` would work here, right? I'd prefer that because comparing the markers is a bit indirect; it makes it sound like `getCursorAtScreenPosition` might return a different `Cursor` instance.